### DISCUSSION TITLE
SCT-013 Support

### DISCUSSION THREAD

**Discussion by RichieRogers** [arendst/Tasmota#14077] [Q&A] [answered]
Hi,
I have a Wemos D1 with a SCT-013 and "shield" that is currently using https://github.com/alcar21/WemosEM.
This app is no longer updated (tbh, I wasn't really happy with it anyway). I'm wondering if I can flash the Wemos D1 with Tasmota and will it work "out of the box"?
I'm using (and am very happy with) Tasmota on all my other devices, so I really want this device brought inline with the others.
Can anyone advise on this or offer another solution that uses this SCT-013 (240v up to 100A clamp sensor)?

Update: the more I look at this the more I think I might be better just buying a PZEM 004T, clamp sensor and hook it up to the Wemos or a NodeMCU I have - I could be wrong, but it seems that the PZEM 004T will read all the values (unlike the SCT-013, which needs to be "told" the voltage).

Thanks,
Richie

**Comment by Jason2866** [comment, marked as answer]
See https://tasmota.github.io/docs/Commands/#sensors

**Reply by RichieRogers** [reply]
Hi, thanks. Just looking at that now, but I'm seriously considering going over to a PZEM 004T, which seems far more accurate.
Thanks,
Richie

**Reply by Jason2866** [reply]
PZEM is very precise and cheap
